experimental/packages/opentelemetry-instrumentation-grpc/src/enums/AttributeNames.ts

```diff
@@ -17,9 +17,26 @@
 /**
  * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md
  */
-export enum AttributeNames {
-  GRPC_KIND = 'grpc.kind', // SERVER or CLIENT
-  GRPC_METHOD = 'grpc.method',
-  GRPC_ERROR_NAME = 'grpc.error_name',
-  GRPC_ERROR_MESSAGE = 'grpc.error_message',
+import { SemanticAttributes } from '@opentelemetry/semantic-conventions';
+
+interface AttributesType {
+  GRPC_KIND: string;
+  RPC_SYSTEM: string;
+  GRPC_METHOD: string;
+  RPC_SERVICE: string;
+  GRPC_ERROR_NAME: string;
+  GRPC_ERROR_MESSAGE: string;
 }
+
+const RPC_METHOD  = SemanticAttributes.RPC_METHOD;
+const RPC_SYSTEM  = SemanticAttributes.RPC_SYSTEM;
+const RPC_SERVICE  = SemanticAttributes.RPC_SERVICE;
+
+export const AttributeNames: Readonly<AttributesType> = {
+  GRPC_KIND: 'grpc.kind', // SERVER or CLIENT
```

Comment:
I know this is not new, but these attribute names are not in the same format as the one defined in the spec: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/rpc.md#grpc-attributes.

Reply:
@legendecas 
Regarding the link you sent, should I just remove this attr (**GRPC_KIND: 'grpc.kind'**)?

Reply:
I'd agree to remove this attribute as it duplicates with the span kind.